valhalla/sif/dynamiccost.h

```diff
@@ -171,6 +172,7 @@ class DynamicCost {
    * @return Returns true if access is allowed, false if not.
    */
   virtual bool Allowed(const baldr::DirectedEdge* edge,
+                       const bool is_dest,
```

Comment:
why not add this as the last argument in the method?

Reply:
This is going after `edge` because it is the property of this edge. It seems more readable if they are near each other. I think, it would be a little strange if I put it after timezone parameter.